### PR TITLE
Tools added

### PR DESCRIPTION
**Pull request by Nishchit14**
Firecamp and GraphiQL added

### CODE REVIEW

**Comment by andimarek** [comment]
@Nishchit14 Thanks for the PR, but we only list tools here that depend/relate to graphql java. I don't think what you added fits into this category. What do you think?

**Comment by Nishchit14** [comment]
I get the reference from here, https://github.com/graphql-java/graphql-java/pull/1843#issuecomment-608988559

Tools category will never be specific to any language, But ya tools might have the support of language., like `code-generation` for java clients. a feature kinda thing.

still, it's all up to you maintainers as you have more clearer plan for it so :D

I have added it so the JAVA community will have visibility on available GraphQL playgrounds which helps them to be more managed and productive while GraphQL development.

File: README.md [hunk 1]
```diff
@@ -109,7 +109,8 @@
 * [graphql-java-datetime](https://github.com/donbeave/graphql-java-datetime): A set of ISO 33601, RFC 3339 compatible date time scalars for GraphQL Java
 
 ## Tools
-
+* [Firecamp](https://firecamp.io/graphql): The fastest GraphQL playground for altra modern team.
```
Comment (benmccann) [inline README.md:112]: I think you mean "ultra" instead of "altra"?

**Comment by benmccann** [comment]
I tend to agree it's not a good fit for this repo. If it were going to be included, I would probably create a new section

File: README.md (hunk 1 above)
Comment (Nishchit14) [reply README.md:112]: Oppss... you're right

**Comment by Nishchit14** [comment]
Sure, no worries @benmccann, you can change it accordingly.

**Comment by andimarek** [comment]
closing  ...see discussion above

**Pull request closed without merging**